src/server/game.js

```diff
@@ -150,13 +150,6 @@ class Game {
 
     if (returnedSpell.isCastingDone()) {
       this.animateAndDealDamage(player);
-      returnedSpell.calculateDamage();
```

Comment:
Nice, konecne je to v takej podobe v akej som tom zamyslal pred refactorom, ktory som nikdy nedokoncil :)

Reply:
nezostalo to tu tak iba kvoli nejakemu mergu, ktory ponechal aj refaktor aj odstranene riadky?